app/models/miq_expression.rb

```diff
@@ -1711,12 +1711,11 @@ def self.is_plural?(field)
 
   def self.atom_error(field, operator, value)
     return false if operator == "DEFAULT" # No validation needed for style DEFAULT operator
+    return _("Ruby scripts in expressions are no longer supported.") if operator.to_s.casecmp("ruby") == 0
```

Comment:
Ignore that one... maybe we should review that rule on the bot.  I prefer the first in terms of readability.  Maybe casecmp is faster but we don't generally trade readability for speed.